src/dune_rules/dune_file.mli

```diff
@@ -330,24 +330,35 @@ module Include_subdirs : sig
     | Include of qualification
 end
 
+module Library_redirect : sig
+  type 'old_name t =
+    { project : Dune_project.t
+    ; loc : Loc.t
+    ; old_name : 'old_name
+    ; new_public_name : Loc.t * Lib_name.t
+    }
+
+  module Private : sig
+    type nonrec t = (Loc.t * Lib_name.Local.t) t
+
+    val old_public_name : t -> Lib_name.t
```

Comment:
This "old_public_name" is not really public, is it? Should we rename it to just old_name?

Reply:
Done.

Reply:
Actually, I just inlined this call. If the name isn't clarifying anything, this one liner function shouldn't even exist.